packages/amplify_core/lib/src/config/amplify_outputs/data/data_outputs.dart

```diff
@@ -38,6 +43,13 @@ class DataOutputs
   /// List of supported authorization types for AWS AppSync.
   final List<APIAuthorizationType> authorizationTypes;
 
+  @override
+  ApiType get apiType => ApiType.graphQL;
```

Comment:
*Question:* Does this get included in the json serialization/de-serialization? I don't see the generated file changed, but that could just be caused by codegen not being run

Reply:
code gen does not change the generated codes, I think this is because they are getters.